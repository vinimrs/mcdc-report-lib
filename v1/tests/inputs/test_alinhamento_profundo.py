def decision(a, b, c, d, e):
  if (a and (b or (c and d))) or e:
      return True
  return False