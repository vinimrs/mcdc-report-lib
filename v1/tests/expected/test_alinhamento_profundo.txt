Relatório de Testes MC/DC
==============================

Decisão: if a and (b or (c and d)) or e
Condições: a, b, c, d, e

Casos de Teste MC/DC:
--------------------
Teste 1: a=False | b=True  | c=True  | d=True  | e=False | Resultado: False
Teste 2: a=True  | b=False | c=False | d=True  | e=False | Resultado: False
Teste 3: a=True  | b=False | c=True  | d=False | e=False | Resultado: False
Teste 4: a=True  | b=False | c=True  | d=False | e=True  | Resultado: True
Teste 5: a=True  | b=False | c=True  | d=True  | e=False | Resultado: True
Teste 6: a=True  | b=True  | c=True  | d=False | e=False | Resultado: True
Teste 7: a=True  | b=True  | c=True  | d=True  | e=False | Resultado: True

==============================
